include/blackhole/formatter/string.hpp

```diff
@@ -191,11 +178,6 @@ class string_t : public formatter_t {
     explicit string_t(const std::string& pattern);
     explicit string_t(const std::string& pattern, severity_map sevmap);
 
-    /// Configuration.
-
-    auto set(const std::string& name, const string::optional_t& option) -> void;
```

Comment:
This is an API breakabe. But despite of I doublt that someone used it a notice in changelog should mention this.